plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1418,19 +1475,21 @@ function( $subscription ) use ( $site_id ) {
 		
 		foreach ( $subscriptions as &$subscription ) {
 			$subscription['active'] = in_array( $site_id, $subscription['connections'] );
-
-			$subscription['local'] = array(
+			$subscription['local']  = array(
 				'installed' => false,
 				'active'    => false,
 				'version'   => null,
 			);
 
 			$updates = WC_Helper_Updater::get_update_data();
-			$local   = wp_list_filter( array_merge( $woo_plugins, $woo_themes ), array( '_product_id' => $subscription['product_id'] ) );
 
+			$local            = wp_list_filter(
+				array_merge( $local_plugins, $local_themes ),
+				array( 'slug' => $subscription['zip_slug'] )
```

Comment:
Note for my future self: picks out the local plugin or theme whose slug matching the zip slug of a subscription. This identifies it as a Woo owned plugin, even if it has no `Woo` header.